semver/mod.ts

```diff
@@ -274,6 +274,14 @@ export interface Options {
   includePrerelease?: boolean;
 }
 
+export interface SemVer {
```

Comment:
How about calling the interface `Version`?

Reply:
Considering its coming from the "semver" module named after the "semver" spec, I felt like this was a pretty nice name while Version is kind of vague.

Also, theoretically there are some other version specs that may be worth supporting eventually such as Python, Nuget, Windows versions etc. It may be worth keeping it the name more specific to give it room.

Reply:
And by support, I mean there may be some situations where parsing other version types and having a method for normalizing them into SemVer may be a good idea, and possibly formatting back out to those other formats.

Reply:
I agree with you for the same reasons, @justinmchase.